[CF1] Access and tunnels with public hostnames callout (#23530)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/connections/connect-networks/routing-to-tunnel/index.mdx

title: Public hostnames
sidebar:
  order: 8
---

With Cloudflare Tunnel, you can expose your HTTP resources to the Internet via a public hostname. For example, you can add a route that points `docs.example.com` to `localhost:8080`. Anyone can now view your local application by going to `docs.example.com` in their web browser.

Cloudflare can route traffic to your Cloudflare Tunnel connection using a [DNS record](/cloudflare-one/connections/connect-networks/routing-to-tunnel/dns/) or [Cloudflare Load Balancer](/cloudflare-one/connections/connect-networks/routing-to-tunnel/lb/). You can configure either option from the Cloudflare dashboard by pointing a DNS `CNAME` record or a load balancer pool to the Cloudflare Tunnel subdomain for your connection. You can also associate these records with your tunnel from `cloudflared` directly.

:::note

> You do not need a paid Cloudflare Access plan to expose a public hostname via Cloudflare Tunnel. A Cloudflare Tunnel can route traffic to a public hostname without requiring an Access. Access licensing is only required if you want to [secure your public hostname with Cloudflare Access](/cloudflare-one/applications/non-http/infrastructure-apps/) rules, such as requiring users to log in via an identity provider, or protect the hostname with Access policies and identity-based authentication.

Reviewer: Can we make it more clear that "securing your public hostname" is still possible on a Free account?

> You do not need a paid Cloudflare Access plan to expose a public hostname via Cloudflare Tunnel. A Cloudflare Tunnel can route traffic to a public hostname without requiring an Access. Access licensing is only required if you want to [secure your public hostname with Cloudflare Access](/cloudflare-one/applications/non-http/infrastructure-apps/) rules, such as requiring users to log in via an identity provider, or protect the hostname with Access policies and identity-based authentication.

Author: ```suggestion
You do not need a paid Cloudflare Access plan to expose a public hostname via Cloudflare Tunnel. A Cloudflare Tunnel can route traffic to a public hostname without requiring a paid Cloudflare Access plan. Access licensing is only required if you want to [secure your public hostname with Cloudflare Access](/cloudflare-one/applications/non-http/infrastructure-apps/) rules, such as requiring users to log in via an identity provider, or protect the hostname with Access policies and identity-based authentication.
```

> You do not need a paid Cloudflare Access plan to expose a public hostname via Cloudflare Tunnel. A Cloudflare Tunnel can route traffic to a public hostname without requiring a paid Cloudflare Access plan. Access licensing is only required if you want to [secure your public hostname with Cloudflare Access](/cloudflare-one/applications/non-http/infrastructure-apps/) rules, such as requiring users to log in via an identity provider, or protect the hostname with Access policies and identity-based authentication.

Reviewer: ```suggestion
You do not need a paid Cloudflare Access plan to publish an application via Cloudflare Tunnel. [Access seats](/cloudflare-one/identity/users/seat-management/) are only required if you want to [secure the application using Access policies](/cloudflare-one/applications/configure-apps/self-hosted-public-app/), such as requiring users to log in via an identity provider.
```

Reviewer: or:
```
You do not need a paid Cloudflare Access plan to publish an application via Cloudflare Tunnel. [Access seats](/cloudflare-one/identity/users/seat-management/) are only required if you want users to [authenticate to the application](/cloudflare-one/applications/configure-apps/self-hosted-public-app/) through Cloudflare Access.
```
